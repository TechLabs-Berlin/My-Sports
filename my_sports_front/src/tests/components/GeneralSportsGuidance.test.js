import React from "react";
import { shallow } from "enzyme";
import GeneralSportsGuidance from "../../components/GeneralSportsGuidance";
import "../setupTests";

test("should render general sports guidance correctly", () => {
  const wrapper = shallow(<GeneralSportsGuidance />);
  expect(wrapper.debug()).toMatchSnapshot();
});
