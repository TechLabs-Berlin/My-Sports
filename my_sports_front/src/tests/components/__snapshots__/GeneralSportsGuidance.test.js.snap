// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render general sports guidance correctly 1`] = `
"<div>
  <h2 className=\\"terms-header text-center mb-5\\">
    General
  </h2>
  <p className=\\"text-center\\">
    1 star - No competetive experience, I might not know the rules, and I don’t care, I just want to have fun.
  </p>
  <p className=\\"text-center\\">
    2 star - A little, not a lot, I have some basic understanding but I’m not very good, but thats ok!
  </p>
  <p className=\\"text-center\\">
    3 star - I know the rules and I have a little experience. I’m up for some light competition and a friendly game.
  </p>
  <p className=\\"text-center\\">
    4 star - I want to win. I have the energy and the skills to do my best and make a good and fair game.
  </p>
  <p className=\\"text-center\\">
    5 star - Go hard or go home. I take this sport seriously and I want to play with those at the highest level.
  </p>
</div>"
`;
